### DISCUSSION TITLE
X11IRAF installation on macOS m1 ventura 13.0.1

### DISCUSSION THREAD

**Discussion by RMnaorch** [iraf-community/iraf#239] [Q&A] [answered]
Hello,
I am installed IRAF follow instructions from the [page](https://iraf-community.github.io/install.html),and there were no proplems.
Installation is done on Macbook pro (13.3-inch, M1, 2020) macOS Ventura version 13.01

However, when I tried installing X11IRAF ,on the step $ make  , and I got the following error:
```
- /Library/Developer/CommandLineTools/usr/bin/make -C obm
cc -I/usr/include/tcl -I/usr/X11/include   -c -o Obm.o Obm.c
cc -I/usr/include/tcl -I/usr/X11/include   -c -o client.o client.c
client.c:137:53: warning: passing 'char ***' to parameter of type 'const char ***' discards qualifiers in nested pointer types [-Wincompatible-pointer-types-discards-qualifiers]
literal:    if (Tcl_SplitList (tcl, command, &argc, &argvp) == TCL_OK) {
/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/tclDecls.h:1453:21: note: passing argument to parameter 'argvPtr' here
                                CONST84 char ***argvPtr);
...
...
2 warnings generated.
cc -I../obm -I/usr/include/tcl -I/usr/X11/include   -c -o zscale.o zscale.c
cc -L/usr/X11/lib -o ximtool eps.o fitsio.o gifio.o iis.o irafio.o ism.o ism_wcspix.o load.o logo.o print.o quant.o rasio.o raster.o save.o tiffio.o util.o ximclient.o ximtool.o zscale.o \
	    -L../obm -lobm \
	    -lXpm -ltcl -lXaw3d -lXmu -lXt -lXext -lX11 -lncurses -lm
cd clients && mkpkg || true
/bin/sh: mkpkg: command not found
```
Does anyone know on how to solve this?
Thanks

**Comment by olebole** [comment, marked as answer]
The **mkpkg** program (which is part of IRAF) must be in the PATH. It is however only needed if you want to use **ximtool** and need the WCS coordinated displayed in the tool. If you use **ds9** as image display, this can safely ignored.

**Reply by RMnaorch** [reply]
Thanks your quickly replay, I want to use the xgterm, but when  I type xgterm and get the error:
```
(base) liulinjie@liulinjiedeMacBook-Pro x11iraf-2.1 % xgterm
zsh: command not found: xgterm
```

**Reply by olebole** [reply]
Did you do a `make install` resp. (if you are not root) a `sudo make install`? If not, yout should put the **xgterm** program (in the `xgterm/` subdirectory) to a place specified in your PATH.

BTW: I would strongly recommend not to start doing IRAF nowadays as long as there is no hard need to. IRAF is very old and deprecated software without institutional support. Please have a look into [Astropy](https://astropy.org) and its [affiliated packages](https://www.astropy.org/affiliated/index.html#coordinated-package-list) to see whether they can be used instead.

**Reply by RMnaorch** [reply]
I just do a `make install` , now after I do the `sudo make install`, the problem was fixed.
Thank you very much. 
Again ,Thanks for your suggestion，I am considering to learn the Astropy.
